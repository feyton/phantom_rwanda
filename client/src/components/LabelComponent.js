import React from 'react';

const LabelComponent = ({ name, htmlFor }) => {
  return (
    <label
      className="block text-grey-darker text-sm font-bold font-rale  mb-2"
      htmlFor={htmlFor}
    >
      {name}
    </label>
  );
};

export default LabelComponent;
