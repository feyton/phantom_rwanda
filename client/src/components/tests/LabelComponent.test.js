import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import LabelComponent from '../LabelComponent.js';

describe('LabelComponent', () => {
  it('should render the LabelComponent component', () => {
    const elem = renderer
      .create(
        <MemoryRouter>
          <LabelComponent />
        </MemoryRouter>
      )
      .toJSON();
    expect(elem).toMatchSnapshot();
  });
});
