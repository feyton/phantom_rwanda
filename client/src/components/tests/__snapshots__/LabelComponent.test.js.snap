// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabelComponent should render the LabelComponent component 1`] = `
<label
  className="block text-grey-darker text-sm font-bold font-rale  mb-2"
/>
`;
